cg/apps/demultiplex/sample_sheet/create.py

```diff
@@ -12,33 +16,44 @@
 LOG = logging.getLogger(__name__)
 
 
+def sample_sheet_creator_factory(
```

Comment:
Maybe `get_sample_sheet_factory` or `get_sample_sheet_creator` is a better name for this function. The current name sounds like a class